tests/e2e/project/commit.spec.ts

```diff
@@ -0,0 +1,83 @@
+import { test, expect } from "@tests/support/fixtures.js";
+
+const sourceBrowsingFixture =
+  "/seeds/0.0.0.0/rad:git:hnrkgd7sjt79k4j59ddh11ooxg18rk7soej8o";
+const modifiedFileFixture = `${sourceBrowsingFixture}/remotes/hyy1k6ggg45pi7ip7ksyn1wt1ob4w5zh1awtg4qu3cxmbh5mws8pj1/commits/0be0f0302269b362be0bfe72aa4843eceaac5e3f`;
+
+test("navigation from commit list", async ({ page }) => {
+  await page.goto(sourceBrowsingFixture);
+  await page.getByTitle("Change peer").click();
+  await page.locator("text=bob hyy1k6").click();
+  await page.locator('role=button[name="Commit count"]').click();
+
+  await page.locator("text=Update readme").click();
+  await expect(page).toHaveURL(modifiedFileFixture);
+});
+
+test("modified file", async ({ page }) => {
+  await page.goto(modifiedFileFixture);
+
+  // Commit header.
+  {
+    const header = page.locator(".commit header");
+    await expect(header.locator("text=bob committed 7 days ago")).toBeVisible();
```

Comment:
This seems to have a difference in the `X days ago` while we expect 7 we show 8 days.
<img width="286" alt="Bildschirm­foto 2022-11-29 um 10 58 26" src="https://user-images.githubusercontent.com/7912302/204498157-dabeb699-9147-4633-9804-cd71b4cff8a9.png">

Not sure yet why.

Reply:
I think I forgot to add time mocking when I moved the tests to a separate file. I wonder why it didn't fail on CI though. 🤔 

Will fix that.

Reply:
Oh, I know why it worked. Because I pushed it yesterday. 🤣

Reply:
d7df79dca4db0cef4de830563366373a32a01c32

Reply:
d7df79d fixed this